README.md

```diff
@@ -300,6 +302,31 @@ plugins:
       visitorData: "paste your visitor_data here"
 ```
 
+## Using a remote cipher server
+
+It becomes harder and harder to keep up with YouTube's cipher changes, as they become more frequent and complex.
+To help with this, you can use a remote cipher server to handle signature deciphering for you.
+You can use [yt-cipher](https://github.com/kikkia/yt-cipher), which is a simple Deno server that exposes a REST API for deciphering signatures.
+Check out the repository for more information on how to set it up.
+
+If you want to implement your own, you can follow the [yt-cipher API specification](https://github.com/kikkia/yt-cipher#api-specification).
+
+### Lavaplayer
+```java
+YoutubeSourceOptions options = new YoutubeSourceOptions()
+    .setRemoteCipherUrl("http://localhost:8001") // The base URL of your remote cipher server.
+    .setRemoteCipherPass("your_secret_password"); // The password to authenticate with your remote cipher server.
+YoutubeAudioSourceManager sourceManager = new YoutubeAudioSourceManager(options, ...);
+```
+
+### Lavalink
+```yaml
+plugins:
+  youtube:
+    remoteCipherUrl: "http://localhost:8001" # The base URL of your remote cipher server.
```

Comment:
Perhaps just have this as a nested block?

```yaml
remoteCipher:
  host: "..."
  auth: "..."
```

Reply:
I think i prefer `url` + `password`